snowflake/table_snowflake_resource_monitor.go

```diff
@@ -0,0 +1,92 @@
+package snowflake
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableSnowflakeResourceMonitor(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "snowflake_resource_monitor",
+		Description: "Lists all the resource monitors in your account for which you have access privileges.",
+		List: &plugin.ListConfig{
+			Hydrate: listSnowflakeResourceMonitors,
+		},
+		Columns: snowflakeColumns([]*plugin.Column{
+			{Name: "name", Type: proto.ColumnType_STRING, Description: "Name for warehouse."},
+			{Name: "credit_quota", Type: proto.ColumnType_DOUBLE, Description: "Monthly credit quota for the resource monitor."},
+			{Name: "used_credits", Type: proto.ColumnType_DOUBLE, Description: "Number of credits used in the current monthly billing cycle by all the warehouses associated with the resource monitor."},
+			{Name: "remaining_credits", Type: proto.ColumnType_DOUBLE, Description: "Number of credits still available to use in the current monthly billing cycle."},
+			{Name: "level", Type: proto.ColumnType_STRING, Description: "Level"},
+			{Name: "frequency", Type: proto.ColumnType_STRING, Description: "Daily, Weekly, etc"},
```

Comment:
Similar to the `level` column, can you please add a bit more to this description as well?